x-pack/plugins/siem/public/pages/hosts/translations.ts

```diff
@@ -15,7 +15,7 @@ export const NO_AUDITBEAT_INDICES = i18n.translate('xpack.siem.hosts.noAuditBeat
 });
 
 export const KQL_PLACE_HOLDER = i18n.translate('xpack.siem.hosts.kqlPlaceHolder', {
-  defaultMessage: 'Search… (e.g. host.name:"foo" AND process.name:"bar")',
+  defaultMessage: 'e.g. host.name: "foo"',
```

Comment:
I wonder if there's a not-so-hidden opportunity here to show off useful domain/task specific queries as the  KQL placeholder. Kind of like a QOTD (query of the day) where it would show a random _insightful_ query to run. Does Kibana core do anything like this?

(I know I would've benefited from a `host.name:*` placeholder when I first started... ^_^)

Reply:
This is a great idea! Probably out of scope for this PR, but I'll cut a ticket for it as a future enhancement.